.circleci/config.yml

```
@@ -4,6 +4,29 @@ orbs:
   win: circleci/windows@2.4.0
 
 jobs:
+  format:
+    docker:
+      - image: ubuntu:18.04
+    steps:
+      - checkout
+      - run:
+          name: Install clang-format
+          command: |
+            apt-get update
+            apt-get install -y git-core clang-format-10
+      - run:
+          name: Verify clang-format
+          command: |
+             git ls-files | grep -E  '\.[cpp|h|cu|cuh]$' | xargs clang-format-10 -i
```

This regex is likely doing something else than intended (TL;DR: `()` instead of `[]`). It sounds like you want to capture all files with extensions `.cpp`, `.h`,  `.cu`, and `.cuh`, but you're matching all extensions made of a single letter that's either `c`, `h`, `p` or `u`.

```suggestion
             git ls-files | grep -E  '\.(cpp|h|cu|cuh)$' | xargs clang-format-10 -i
```

---

Good catch, thanks for flagging!